platform/darwin/src/MGLStyleValue.mm

```diff
@@ -62,16 +62,59 @@ - (NSUInteger)hash {
 
 @end
 
+@implementation MGLStyleFunction
+
++ (instancetype)functionWithStops:(NSDictionary *)stops {
+    return [[self alloc] initWithInterpolationBase:1.0 stops:stops];
+}
+
++ (instancetype)functionWithInterpolationBase:(CGFloat)interpolationBase stops:(NSDictionary *)stops {
+    return [[self alloc] initWithInterpolationBase:interpolationBase stops:stops];
```

Comment:
Can we initialize an MGLCameraStyleFunction here instead (and in `+functionWithStops:`, but not `-init` or `-initWithInterpolationBase:stops:`)? If there are warnings associated with the designated initializer, we can suppress the warnings.